nym-vpn-desktop/src-tauri/Cargo.toml

```diff
@@ -32,6 +32,9 @@ reqwest = { version = "0.11", features = ["json"] }
 itertools = "0.12"
 clap = { version = "4.5", features = ["derive"] }
 build-info = "0.0.34"
+sled = "0.34.7"
+strum = { version = "0.26", features = ["derive"] }
+tap = "1.0.1"
```

Comment:
this can be now replaced with `inspect_err` from standard library : )

Reply:
nice one, didnt know about it, thx ;)